### PR TITLE
Chapter1/homework/3

### PR DESCRIPTION
**Pull request by Zacc806**
Reverse engineering the testfile proved to work for me. Sorry, if that wasn't intended

### CODE REVIEW

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+using System.Text;
+using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: Missing something here 😄

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            
+        }
+        public static int PromptInt(string message)
+            {
+                Console.Write(message);
+                int intedMessage = int.Parse(Console.ReadLine());
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: You're not getting a stringed message- you're getting a number.

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+                return intedMessage;
+            }
+            
+        public static string PromptString(string message)
+            {
+                Console.Write(message);
+                var stringedMessage = Console.ReadLine();
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: You're not getting a stringed message- you're getting a string.

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+                return stringedMessage;
+            }
+            
+        public static float PromptFloat(string message)
+            {
+                Console.Write(message);
+                float floatedMessage = float.Parse(Console.ReadLine());
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: You're not getting a stringed message- you're getting a number.

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+                return floatedMessage;
+            }
+
+            
+        public static float calculateBmi(float W, float H)
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: please don't use one character variable names, as it makes code unclear. Also use camelCase and not PascalCase, even for abreviations, when passing variables.

**Review by Almantask — CHANGES_REQUESTED** [review]
Please note to what branch are you trying to merge. You should have aimed at Chapter1/Homework/3 and not master (to the same one you are at, except the original repo)

File: Src/BootCamp.Chapter/Lesson3.cs (hunk 2 above)
Comment (Zacc806) [reply Src/BootCamp.Chapter/Lesson3.cs]: I can't get my head around it. Aren't i getting a message (in string) for requesting to enter a number and then parsing the input into int?

Comment (Almantask) [reply Src/BootCamp.Chapter/Lesson3.cs]: Console.ReadLine()- this is getting the user input
int.Parse() - this is turned that to an int
The result?
You have a number.

Comment (Zacc806) [reply Src/BootCamp.Chapter/Lesson3.cs]: image

Comment (Zacc806) [reply Src/BootCamp.Chapter/Lesson3.cs]: > Console.ReadLine()- this is getting the user input
> int.Parse() - this is turned that to an int
> The result?
> You have a number.
My code does exactly that. 
Pormpting int(age) asks for age, int.parse parses the readlined number and it does give number.

File: Src/BootCamp.Chapter/Lesson3.cs (hunk 1 above)
Comment (Zacc806) [reply Src/BootCamp.Chapter/Lesson3.cs]: what shall i write here? Messages prompting for name, surname, age and height/weight?

Comment (Almantask) [reply Src/BootCamp.Chapter/Lesson3.cs]: Yes, as the name implies, demo that the method works.

Comment (Zacc806) [reply Src/BootCamp.Chapter/Lesson3.cs]: Fixed it and all the other problems, Should be fine now

**Review by Almantask — APPROVED** [review]
Looks good, just be careful to what branch are you merging.

**Pull request closed without merging**